views/default/elgg/menus/dropdown.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Dropdown menu module
+ *
+ * @module elgg/menus/dropdown
+ */
+define(function (require) {
+
+	var elgg = require('elgg');
+	var popup = require('elgg/popup');
+
+	var dropdown = {
+
+		init: function () {
+			$(document).on('click', '.elgg-menu-item-has-dropdown > a', function (e) {
+				var $trigger = $(this);
+				if ($trigger.data('dropdownMenu')) {
+					var $target = $trigger.data('dropdownMenu');
+				} else {
+					var $target = $trigger.siblings('.elgg-child-menu').eq(0);
+					$trigger.data('dropdownMenu', $target);
+
+					$target.on('open', function () {
```

Comment:
If $target is an empty jQuery container this should have no effect, so this is fine as is, right?